ref/feature/mpJwt/examples.adoc

```diff
@@ -1,35 +1,33 @@
-The `mpJwt` feature adds the libraries that are required for MicroProfile JWT implementation.
+== Examples
+
+=== Basic configuration to implement MicroProfile JWT
 
 The `<mpJwt/>` element is required to configure the processing of the caller’s JWT.
 
+The Secure Sockets Layer(SSL)and the JSON Web Token(JWT) uses a keystore by using the maven keytool plugin.The keystore is built by using the maven keytool plugin.The applicaton uses HTTP ports and contains the user and login endpoints. The MP JWT configuration injects the caller's JWT into a ResourceScoped bean for inspection.
```

Comment:
`The Secure Sockets Layer(SSL)and the JSON Web Token(JWT) uses a keystore by using the maven keytool plugin.`
- add space between ) & and
- "uses" should be "use" since subject is plural
- I'm not sure what this means- SSL is a protocol, so it doesn't necessarily "use" anything- Not sure it needs to be explained in this context but the relationship between keystore and keytool plugin is not clear.  Does JWt access the keystore by using the plugin?
- need a space between period and "The"  `plugin.The keystore`

`The keystore is built by using the maven keytool plugin.`

Does this mean- The Maven keytool plugin creates the keystore if one does not already exist? might be a question for SME- more specifically- why does the keystore matter to JWT config?


`The applicaton uses HTTP ports and contains the user and login endpoints.`
- application (sp)
- I'm not sure what this means. It might be helpful to call out the elements (eg. `httpEndpoint`) in the description so the user can easily connect them to the config example

Are the Keystore, webapp, http endpoint, etc part of the MPjwt config?  They are explained but not put in context- and the attributes of the mp-jwt element aren't explained